content/articles/tidyverse_1.2.0.Rmd

```diff
@@ -0,0 +1,75 @@
+---
+title: tidyverse 1.2.0
+slug: tidyverse-1.2.0
+description: >
+  A new version of the tidyverse package is now on CRAN.
+author: Mara Averick
+date: 2017-11-15
+photo:
+  url: https://pixabay.com/en/universe-sky-star-space-all-1566159/
+  author: Gerd Altmann
+categories: [package]
+---
+
+```{r setup, include = FALSE}
+library("tidyverse")
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+The release of [tidyverse 1.2.0](https://github.com/tidyverse/tidyverse/releases) 
+on CRAN (now 1.2.1, following some minor fixes) includes updates in tidyverse 
+package membership, as well as to tidyverse functions themselves.
+
+The latest version of tidyverse can be installed with:
+
+```{r tidyverse_install, eval = FALSE}
+install.packages("tidyverse")
+```
+
+## Changes in tidyverse membership
+
+### Addition of the reprex package
+
+Though it's been a mainstay tidyverse [Help](https://www.tidyverse.org/help/), 
+1.2.0 marks the official addition of [reprex](http://reprex.tidyverse.org/) 
+(short for **repr**oducible **ex**ample) to the tidyverse. 
+
+The goal of a reprex is to package your problematic code in such a way that 
+other people can run it and feel your pain. Think of every request for a 
+reprex as a plea to _"help me help you!"_
+
+This update includes reprex **Addins**, accessible from the RStudio drop-down 
+menu.
+
+![](https://i.imgur.com/PxRzbTq.png)
+
+### Updates to tidyverse core
+
+The [stringr](http://stringr.tidyverse.org/) and [forcats](http://forcats.tidyverse.org/)
+packages (for string, and categorical variable manipulation, respectively) have 
+joined [ggplot2](http://ggplot2.tidyverse.org/), [dplyr](http://dplyr.tidyverse.org/), [tidyr](http://tidyr.tidyverse.org/), [readr](http://readr.tidyverse.org/), 
+[purrr](http://purrr.tidyverse.org/), and [tibble](http://tibble.tidyverse.org/) 
+to make up the core tidyverse. 
+
+Running `library(tidyverse)` attaches these core packages in the current R session.
+On attach, tidyverse now makes better use of horizontal space, printing packages
+and versions in two columns (see below). Additionally, packages that have 
+already been attached are not be printed, and development versions are highlighted
+in red. The startup message can now be suppressed with 
+`options(tidyverse.quiet = TRUE)`.
+
+![](https://i.imgur.com/y2VcJ24.gif)
+
+
+## Other improvements
+
+While a comprehensive list of changes can be found in the [release notes](https://github.com/tidyverse/tidyverse/releases), other highlights 
+include:
+
+ * `tidyverse_conflicts()` now prints all conflicts involving at least one
+ tidyverse package (including intra-tidyverse conflicts, omitted in previous
+ versions)
+ 
+ * Addition of a `tidyverse_logo()` function.
+
+![](https://i.imgur.com/D6TJvPn.gif)
```

Comment:
Can you please also include an acknowledgements section that thanks:

```
[Derek Chiu](https://github.com/dchiu911), [Matthew Lincoln](https://github.com/mdlincoln), [Michael Toth](https://github.com/michaeltoth) and [Serhat Cevikel](https://github.com/serhatcevikel)
```

And I'll share my script with you to use in the future. It probably needs to go into some sort of repo somewhere